.github/workflows/build-python-client.yml

```diff
@@ -0,0 +1,98 @@
+name: Python-client
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - 'python-client/**'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false # Do not stop when any job fails
+      matrix:
+        python-version: [ "3.8", "3.9", "3.10" ]
+        os: [ubuntu-latest]
+        experimental: [false]
+        # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
+        include:
+          # - python-version: "3.11"
+          #   os: ubuntu-latest
+          #   experimental: true
+          # - python-version: "3.10"
```

Comment:
why did you leave it commented out?

Reply:
I was wondering if we should run it for mac os or not.
Same for windows-2022

for python3.11, since I changed the pyproject.toml, no need to run, I can remove it

Reply:
got it, lets not overcomplicate it by adding all the runner types. I type of windows should be fine (I believe 2019 is a more common one these days, right?)
In any case let's clean up the commented out runners